pkg/api/v1alpha1/oidcconfig_webhook.go

```diff
@@ -40,6 +40,14 @@ func (r *OIDCConfig) ValidateUpdate(old runtime.Object) error {
 	if !ok {
 		return apierrors.NewBadRequest(fmt.Sprintf("expected a OIDCConfig but got a %T", old))
 	}
+	oidcconfiglog.Info("Managed vs Workload", "IsManaged()", r.IsManaged())
+
+	if r.IsManaged() {
+		oidcconfiglog.Info("Workload cluster")
```

Comment:
can we do
```suggestion
		oidcconfiglog.Info("OIDC config is associated with workload cluster", "name", old.Name)
```
same thing below